biometricks/src/main/java/com/handstandsam/biometricks/internal/BiometricksHelper.kt

```diff
@@ -78,4 +86,34 @@ internal class BiometricksHelper(context: Context) {
             }
         }
     }
+
+    /**
+     * Check if the user can securely authenticate with biometrics.
+     */
+    fun canSecurelyAuthenticate(): Boolean {
+        if (Build.VERSION.SDK_INT < 28) {
+            // we only have fingerprint, checking if there are any enrolled fingerprints should be enough.
+            val service = (context.getSystemService(Context.FINGERPRINT_SERVICE) as FingerprintManager?)
+                ?: return false
+            return service.hasEnrolledFingerprints()
```

Comment:
Yeah.  My main goal of this library is to try and just help differentiate (as accurately as possible) the capabilities of the device for user messaging/business decisioning.

I get the point for `canSecurelyAuthenticate`, but can't we just force usage to be a certain way so that this will work in specific cases only?  I think shipping a small MVP is more important than solving everything.